src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -450,6 +464,8 @@ DistributedSearchTreeImpl<DeviceType>::queryDispatch(
     ExecutionSpace const &space, Predicates const &queries,
     Callback const &callback, OutputView &out, OffsetView &offset)
 {
+  Kokkos::Profiling::pushRegion("ArborX::DistributedSearchTree::spatial_query");
```

Comment:
Not sure this is a right name.

Reply:
Why not? We also use `ArborX:BVH:spatial_queries`.

Reply:
That's going to change.

Reply:
We could merge this first and then change it consistently for BVH and DST.